gke-deploy/core/resource/resource.go

```diff
@@ -91,47 +91,59 @@ func DecodeFromYAML(ctx context.Context, yaml []byte) (*Object, error) {
 func ParseConfigs(ctx context.Context, configs string, oss services.OSService) (Objects, error) {
 	objs := Objects{}
 
+	if configs == "-" {
+		objs, err := parseResourcesFromFile(ctx, configs, objs, oss)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse config from stdin: %v", err)
+		}
+		return objs, nil
+	}
+
 	fi, err := oss.Stat(ctx, configs)
 	if err != nil {
 		return nil, fmt.Errorf("failed to get file info for %q: %v", configs, err)
 	}
 
-	if fi.IsDir() {
-		d := configs
-		files, err := oss.ReadDir(ctx, d)
-		if err != nil {
-			return nil, fmt.Errorf("failed to list files in directory %q: %v", d, err)
-		}
+	hasResources := false
 
-		hasResource := false
-		for _, fi := range files {
-			if fi.IsDir() {
-				continue
-			}
-			filename := filepath.Join(d, fi.Name())
-			if !hasYamlOrYmlSuffix(filename) {
-				continue
+	var walk func(path string, fi os.FileInfo,  depth int) error
```

Comment:
Not sure if this needs to be declared

Reply:
Unfortunately, it needs to be declared because it's recursive.  I'll add a comment here in my next PR